tests/unit/amo/components/TestWrongPlatformWarning.js

```diff
@@ -42,34 +39,33 @@ describe(__filename, () => {
     });
   };
 
-  const render = ({ location = createFakeLocation(), ...customProps } = {}) => {
+  const render = ({ location, ...customProps } = {}) => {
     const props = {
       _correctedLocationForPlatform,
       _getClientCompatibility,
       _isFirefoxForAndroid,
       _isFirefoxForIOS,
       _isAndroidInstallable,
-      i18n: fakeI18n(),
-      store,
       ...customProps,
     };
+    const renderOptions = {
+      history: createHistory({
+        initialEntries: [location || '/'],
+      }),
+      store,
+    };
 
-    return shallowUntilTarget(
-      <WrongPlatformWarning {...props} />,
-      WrongPlatformWarningBase,
-      {
-        shallowOptions: createContextWithFakeRouter({ location }),
-      },
-    );
+    return defaultRender(<WrongPlatformWarning {...props} />, renderOptions);
   };
 
   it('can add a custom className', () => {
-    _correctedLocationForPlatform.returns('/some/location/');
+    _correctedLocationForPlatform.mockReturnValue('/some/location/');
     const className = 'some-class-name';
-    const root = render({ className });
+    render({ className });
 
-    expect(root).toHaveClassName('WrongPlatformWarning');
-    expect(root).toHaveClassName(className);
+    expect(screen.getByClassName('WrongPlatformWarning')).toHaveClass(
```

Comment:
is there text we can test for rather than the class name?

Reply:
The problem is that the `div` that gets the class added to it is just a wrapper, so it does not itself contain text that we can use to target it.